File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        lookup = {}
        for i, num in enumerate(nums):
            print(i,num)
            complement = target - num
            if complement in lookup:
                return [lookup[complement],i]
            lookup[num] = i
       

File: 1-two-sum/two-sum.go
import "fmt"
func twoSum(nums []int, target int) []int {
    indexPa := make(map[int]int)
    for i, num := range nums{
         
        leftval := target-num
        if j , found :=indexPa[leftval] ; found{
            return []int{j,i}

        }
        indexPa[num] = i
    }
    return nil
    
}